packages/insert-menu/src/InsertMenu.tsx

```diff
@@ -0,0 +1,297 @@
+import {SearchIcon, ThLargeIcon, UlistIcon} from '@sanity/icons'
+import {type SchemaType} from '@sanity/types'
+import {
+  Box,
+  Button,
+  Flex,
+  Grid,
+  Menu,
+  MenuItem,
+  type MenuItemProps,
+  Stack,
+  Tab,
+  TabList,
+  Text,
+  TextInput,
+  Tooltip,
+} from '@sanity/ui'
+import {type ChangeEvent, createElement, useReducer, useState} from 'react'
+import {isValidElementType} from 'react-is'
```

Comment:
Since `react-is` is used here we need to add it to peer deps and dev deps the same way we specify `react` and `react-dom` 🤔

We should probably go over our other packages as I suspect a lot of our npm packages are doing this wrong 😅

Reply:
Ah, of course. I should have caught that because I was actually looking in the `dist` folder to verify that we didn't bundle React and friends, but somehow I missed this one. `dist` is much cleaner now.

Reply:
Fixed: 588a3971f53d1e12073b5f9a8617d7d819bc3d83